modules/auxiliary/gather/magento_xxe_cve_2024_34102.rb

```diff
@@ -0,0 +1,205 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer
+
```

Comment:
Did you intentionally remove this? I think it makes sense to keep.
```suggestion
  include Msf::Exploit::Remote::HttpServer
  prepend Msf::Exploit::Remote::AutoCheck
```